pkg/driver/controller.go

```diff
@@ -444,7 +445,71 @@ func (cs *controller) CreateSnapshot(
 	req *csi.CreateSnapshotRequest,
 ) (*csi.CreateSnapshotResponse, error) {
 
-	return nil, status.Error(codes.Unimplemented, "")
+	klog.Infof("CreateSnapshot volume %s for %s", req.Name, req.SourceVolumeId)
+
+	snapTimeStamp := time.Now().Unix()
```

Comment:
can we add verifySnapshotRequest to verify all the csi compliant things there (see the latest zfs-localpv code https://github.com/openebs/zfs-localpv/blob/master/pkg/driver/controller.go#L581)

Reply:
Done.